km_cli/km_client.c

```diff
@@ -104,7 +104,7 @@ void usage(void)
 {
    fprintf(stderr,
            "Usage: %s [-l] [-c commandname] [-d snapshot_dirname] [-p processid] [-s "
-           "socket_name] [-t]\n",
+           "socket_name] [-t] [-x]\n",
```

Comment:
`-x`?

Reply:
I was considering using the expire flag (x) but realized that didn't make sense.